Source/OxyPlot/Annotations/LineAnnotation.cs

```diff
@@ -96,7 +98,7 @@ protected override IList<ScreenPoint> GetScreenPoints()
                     points.Add(new DataPoint(this.ActualMinimumX, fx(this.ActualMinimumX)));
                     points.Add(new DataPoint(this.ActualMaximumX, fx(this.ActualMaximumX)));
                 }
-                else
+                else if (fy != null)
```

Comment:
Probably ought to throw if both are none, to catch any internal implementation errors.